OpenSourcePolitics/decidim-app
# [Fix] Compatibility between committee promotion and initiatives creation

#### :tophat: Description
*Please describe your pull request.*

#### :pushpin: Related Issues
- [Notion card](https://www.notion.so/opensourcepolitics/Decidim-app-0-27-Compatibilit-AH-et-comit-de-promotion-des-initiatives-d435f3844d87464b892f9bd232017404?pvs=4)

#### Testing

Example:
* Generate your app
* Clear every decidim initiatives type
* Log as an admin
* Create a new one 
* Link an authorization to it
* Add the committee promotions to it
* Create a new initiative
* Try invite yourself with a new account
* See that you are well redirected everywhere you need
* Fill in informations
* See that you're well redirected to the invitation
* Try to send an invitation
* Check that all the process beneath is well completed **_(Don't forget to put your notification digest to Real Time otherwise you won't be able to get the notification instantly)_**

#### Tasks
- [x] Extend the committee invitation controller to handle authorizations


### :camera: Screenshots
*Please add screenshots of the changes you're proposing if related to the UI*
